llpc/context/llpcPipelineContext.cpp

```diff
@@ -254,15 +254,14 @@ void PipelineContext::setPipelineState(Pipeline *pipeline, Util::MetroHash64 *ha
     if (getPreRasterHasGs())
       pipeline->setPreRasterHasGs(true);
   }
-  if (!unlinked) {
-    // Give the shader stage mask to the middle-end. We need to translate the Vkgc::ShaderStage bit numbers
-    // to lgc::ShaderStage bit numbers. We only process native shader stages, ignoring the CopyShader stage.
-    unsigned stageMask = getShaderStageMask();
-    if (hasRayTracingShaderStage(stageMask))
-      stageMask = ShaderStageComputeBit;
-    // Give the user data nodes to the middle-end, and/or hash them.
-    setUserDataInPipeline(pipeline, hasher, stageMask);
-  }
+  // Give the shader stage mask to the middle-end. We need to translate the Vkgc::ShaderStage bit numbers
+  // to lgc::ShaderStage bit numbers. We only process native shader stages, ignoring the CopyShader stage.
+  unsigned stageMask = getShaderStageMask();
+  if (hasRayTracingShaderStage(stageMask))
+    stageMask = ShaderStageComputeBit;
+
+  // Give the user data nodes to the middle-end, and/or hash them.
+  setUserDataInPipeline(pipeline, hasher, stageMask);
```

Comment:
This change results in `unlinked` not being used anymore in this method. Please remove it from the method arguments as an additional cleanup.

Reply:
`unlinked` is still used in `GraphicsContext::setPipelineState`
https://github.com/GPUOpen-Drivers/llpc/blob/dev/llpc/context/llpcGraphicsContext.cpp#L195